README.md

```diff
@@ -3,17 +3,28 @@ A unified platform that integrates smart devices across diverse facilities, empo
 
 ## Project Setup
 
-1. Ensure you have Python 3.13.2 installed on your system. You can download it from [python.org](https://www.python.org/downloads/).
+### Requirements
 
-2. Clone the repository:
+1. [Docker](https://docs.docker.com/get-started/get-docker) - v27.x
+2. [make](https://formulae.brew.sh/formula/make) - v3.x _(Optional)_ ([What is Makefile?](https://opensource.com/article/18/8/what-how-makefile))
```

Comment:
Why link to the homebrew's make formula?
Also, whats' the use of the 2nd link/text????